main.go

```diff
@@ -74,10 +66,5 @@ func main() {
 		l.Fatal("We were told to use the 'any' interface but it's not present.")
 	}
 
-	go func() {
-		if err := http.ListenAndServe(":80", http.HandlerFunc(redirectToTLS)); err != nil {
-			l.Printf("ListenAndServe port 80 error: %v", err)
-		}
-	}()
-	l.Println(http.ListenAndServeTLS(":443", fullChain, privKey, nil))
+	l.Println(http.ListenAndServe(":8443", nil))
```

Comment:
In the case that we're exposing this over HTTP, we will have to test that the zerotrace code that extracts the connection works correctly with the reverse-proxy set up.., [as it extracts the *tls.Conn](https://github.com/brave-experiments/proxy-detection-server/blob/8e460d16ffa70a29662df7ccf6706aa807737180/zerotrace.go#L140)

(and zerotrace connection is made from the clientside using a [secure websocket](https://github.com/brave-experiments/proxy-detection-server/blob/8e460d16ffa70a29662df7ccf6706aa807737180/html_templates.go#L220))

Reply:
Noted! I've done some refactoring already and this doesn't look like a problem.